worlds/musedash/__init__.py

```diff
@@ -215,24 +215,17 @@ def create_items(self) -> None:
                 continue
 
             # Otherwise add a random assortment of songs
-            self.multiworld.random.shuffle(song_keys_in_pool)
+            self.random.shuffle(song_keys_in_pool)
             for i in range(0, needed_item_count - item_count):
                 self.multiworld.itempool.append(self.create_item(song_keys_in_pool[i]))
 
             item_count = needed_item_count
 
     def create_regions(self) -> None:
-        # Basic Region Setup: Menu -> Song Select -> Songs
         menu_region = Region("Menu", self.player, self.multiworld)
         song_select_region = Region("Song Select", self.player, self.multiworld)
-
-        song_select_entrance = Entrance(self.player, "Song Select Entrance", menu_region)
-
-        menu_region.exits.append(song_select_entrance)
-        song_select_entrance.connect(song_select_region)
-
-        self.multiworld.regions.append(menu_region)
-        self.multiworld.regions.append(song_select_region)
+        self.multiworld.regions += [menu_region, song_select_region]
```

Comment:
i really wonder if we should have a helper for region creation that does this too. it'd be technically incorrect to do it in the constructor, though that's what i do in my world. the only place i could think to do this would be `World.create_region` but that feels wrong.